app/mailers/hackathons/digest_mailer.rb

```diff
@@ -32,4 +32,25 @@ def admin_summary(sent_digests)
 
     mail to: Hackathons::SUPPORT_EMAIL, cc: User.admins.collect(&:email_address), subject:
   end
+
+  def organizer_summary(sent_digests)
+    # This reloads the (possible) sent_digests array as an
+    # ActiveRecord::Relation so that we can use includes to prevent an N+1.
+    @sent_digests = Hackathon::Digest.where(id: sent_digests.map(&:id)) ||
+      Hackathon::Digest.where(created_at: 6.days.ago...Time.now)
+
+    @sent_digests_by_hackathons = @sent_digests
+      .includes(listings: {hackathon: {logo_attachment: :blob}})
+      .flat_map(&:listings).group_by(&:hackathon)
+      .transform_values { |listings| listings.map(&:digest).uniq }
+
+    @listed_hackathons = @sent_digests_by_hackathons.keys
+      .sort_by { |hackathon| @sent_digests_by_hackathons[hackathon].count }.reverse!
```

Comment:
For this use case, we don't need to sort the hackathons, so something like this should suffice
```suggestion
    @listed_hackathons = @sent_digests_by_hackathons.keys
```